management/log.go

```diff
@@ -89,6 +89,11 @@ type Log struct {
 
 	// The user's unique identifier
 	UserID *string `json:"user_id"`
+	// The user's name
+	UserName *string `json:"user_name"`
+
+	// Organization Id
+	OrganizationID *string `json:"organization_id"`
```

Comment:
Hey @ashok2sa 👋🏻 Thanks for the contribution! I pushed the generated fields as that was missing from your initial contribution. However, the `organization_id` is not something we expose on the log response https://auth0.com/docs/api/management/v2#!/Logs/get_logs_by_id. May I ask where is this coming from?

Reply:
Hii @sergiught , sorry for late response but I just went over the a log from aurh0 dashboard and i see this log, i needed these information in the log response.  

```{
  "date": "2022-09-30T07:24:38.082Z",
  "type": "fu",
  "description": "Wrong email or password.",
  "connection": "Username-Password-Authentication",
  "connection_id": "con_6qHCIBB1YxxxxxxP",
  "client_id": "xxxxxxxxxxxxxxxxxOC7D1MTx5mY",
  "client_name": "xxxxxx",
  "ip": "2a01:261:xxxxxxxxxxxxxxxx0:e3f0",
  "user_agent": "Chrome 105.0.0 / Linux 0.0.0",
  "details": {
    "error": {
      "message": "Wrong email or password."
    }
  },
  "user_id": "",
  "user_name": "qa-test-ad@xxxxxxx.xx",
  "strategy": "auth0",
  "strategy_type": "database",
  "organization_id": "org_lxxxxxxxxxxx",
  "organization_name": "dev-hxxxxxxxxxxxxxxxx",
  "log_id": "900202209300724395644068205591303666258",
  "_id": "9002022093007243952258247153150303666258",
  "isMobile": false,
  "id": "90020220930072439564406820559165722258247153150303666258"
}```

Reply:
> Hey @ashok2sa 👋🏻 Thanks for the contribution! I pushed the generated fields as that was missing from your initial contribution. However, the `organization_id` is not something we expose on the log response https://auth0.com/docs/api/management/v2#!/Logs/get_logs_by_id. May I ask where is this coming from?

@sergiught I didnt see any new fields in  `log struct` in ```log.go``` file.

Reply:
Many thanks @sergiught !!!